### PR TITLE
Open eBooks Custom Login Page

### PR DESCRIPTION
**Pull request by kristojorg**
This is a custom `LoginPicker` for Open eBooks that shows some documentation of the two available login methods (taken from the home page).

### CODE REVIEW

File: src/auth/LoginPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
```
Comment (kristojorg) [inline src/auth/LoginPicker.tsx:1]: This file is simply extracted from the home page

File: src/auth/OpenEbooksLoginPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+/** @jsx jsx */
```
Comment (kristojorg) [inline src/auth/OpenEbooksLoginPicker.tsx:1]: This file is extracted from `/components/OpenEbooksLanding.tsx`

File: src/auth/LoginPicker.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      >
+        {authMethods?.map(method => (
+          <option key={method.id} value={method.id}>
+            {method.description}
+          </option>
+        ))}
+      </Select>
+      <AuthButton method={selectedMethod} />
+    </div>
+  );
```
Comment (kaplanmichael) [inline src/auth/LoginPicker.tsx:127]: If I click one of the buttons, proceed to its login form, and then click the browser's back button, I see a version of the login page where the buttons do not appear properly and any subsequent activity will fail:
image

image

**Review by kaplanmichael — CHANGES_REQUESTED** [review]

File: src/auth/LoginPicker.tsx (hunk 3 above)
Comment (kristojorg) [reply src/auth/LoginPicker.tsx:127]: This is a symptom of the ip address circ manager error that is being resolved elsewhere

File: src/auth/LoginPicker.tsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+ * Renders buttons that allow selecting between auth providers.
+ */
+const Buttons: React.FC<{
+  authMethods: AppAuthMethod[];
+}> = ({ authMethods }) => {
+  return (
+    <Stack direction="column" aria-label="Available authentication methods">
+      {authMethods.map(method => (
+        <AuthButton key={method.id} method={method} />
+      ))}
```
[the remark this answers is not in the document]
Comment (kristojorg) [reply src/auth/LoginPicker.tsx:91]: Okay so the buttons and the logo were actually meant to be left-aligned according to the design (at this width). So I've updated to do that and I think it looks better. I also fixed some other design stuff like you pointed out. Should be more in line with the design specs now

File: src/auth/LoginPicker.tsx (hunk 3 above)
Comment (kaplanmichael) [reply src/auth/LoginPicker.tsx:127]: Can't wait! :)

File: src/auth/LoginPicker.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (kaplanmichael) [reply src/auth/LoginPicker.tsx:91]: Looks great now!

**Review by kaplanmichael — APPROVED** [review]

**Pull request merged**